deploy/kubernetes/overlays/alpha/rbac_add_snapshots_to_provisioner.yaml

```diff
@@ -0,0 +1,16 @@
+# arrays without strategic patch merge defined need to be appended
+# using jsonpatch
+# https://github.com/kubernetes-sigs/kustomize/blob/master/examples/jsonpatch.md
+- op: add
+  path: /rules/-
+  value:
+    apiGroups: ["snapshot.storage.k8s.io"]
+    resources: ["volumesnapshots"]
+    verbs: ["get", "list"]
+
+- op: add
```

Comment:
these have to be two seperate rules? We can't consolidate them somehow?

Reply:
Based on the syntax, I don't think it's possible to combine them.  The `value` field is not an array